tests/internal/test_views.py

```diff
@@ -1062,3 +1062,117 @@ def test_reset_imported_tenants_excludes_delete(self, delay):
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertIn("Deleted 2 tenants.", logs.output[0])
         self.assertEqual(4, Tenant.objects.count())
+
+    def test_update_system_flag_in_role(self):
+        """Test that we can update a role."""
+        tenant = Tenant.objects.create(tenant_name="1234", org_id="1234")
+        custom_role = Role.objects.create(
+            name="role 1", system=True, tenant=tenant, platform_default=False, admin_default=False
+        )
+
+        request_body = {
+            "system": "false",
+        }
+
+        json_request_body = json.dumps(request_body)
+
+        self.assertEqual(custom_role.system, True)
```

Comment:
nitpick: in this cases you can use `self.assertFalse()` or `self.assertTrue()` too